python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -111,79 +116,22 @@
     '\n'
 )
 
-# FIXME need to support comprehensive error handling here
-
-# ... on PipelineConfigValidationInvalid {
-#     pipeline {
-#         name
-#     }
-#     errors {
-#         message
-#         path
-#         stack {
-#         entries {
-#             __typename
-#             ... on EvaluationStackPathEntry {
-#             field {
-#                 name
-#                 description
-#                 configType {
-#                 key
-#                 name
-#                 description
-#                 }
-#                 defaultValue
-#                 isOptional
-#             }
-#             }
-#         }
-#         }
-#         reason
-#     }
-#     }
-#     ... on StartSubplanExecutionInvalidStepsError {
-#     invalidStepKeys
-#     }
-#     ... on StartSubplanExecutionInvalidOutputError {
-#     step {
-#         key
-#         inputs {
-#         name
-#         type {
-#             key
-#             name
-#         }
-#         }
-#         outputs {
-#         name
-#         type {
-#             key
-#             name
-#         }
-#         }
-#         solid {
-#         name
-#         definition {
-#             name
-#         }
-#         inputs {
-#             solid {
-#             name
-#             }
-#             definition {
-#             name
-#             }
-#         }
-#         }
-#         kind
-#     }
-#     }
-# }
-# }
-
+# TODO need better error handling for PythonError, StartSubplanExecutionInvalidStepsError,
+# StartSubplanExecutionInvalidOutputError, ...?
 
 LINE_LENGTH = 100
 
+
+def mkdir_p(path):
```

Comment:
we have this in dagster.utils

Reply:
Oh right you are copying and pasting that in here to avoid dependencies. We should really deal with that.